tests/com/twitter/intellij/pants/util/PantsUtilTest.java

```diff
@@ -40,17 +31,19 @@ public void testIsPantsProjectFile() {
   }
 
   public void testIsCompatibleVersion() {
-    assertTrue(PantsUtil.isCompatibleVersion("2.2.3", "1.2.3"));
-    assertTrue(PantsUtil.isCompatibleVersion("1.4.4", "1.2.3"));
-    assertTrue(PantsUtil.isCompatibleVersion("1.2.4", "1.2.3"));
-    assertTrue(PantsUtil.isCompatibleVersion("1.2.4rc0", "1.2.3"));
-    assertTrue(PantsUtil.isCompatibleVersion("1.2.4.dev0", "1.2.3"));
-    assertTrue(PantsUtil.isCompatibleVersion("1.2.3", "1.2.3"));
-
-    assertFalse(PantsUtil.isCompatibleVersion("1.2.0rc122", "1.2.3"));
-    assertFalse(PantsUtil.isCompatibleVersion("2.34.43", "2.34.44"));
-    assertFalse(PantsUtil.isCompatibleVersion("2.33.44", "2.34.44"));
-    assertFalse(PantsUtil.isCompatibleVersion("1.34.44", "2.34.44"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("2.2.3", "1.2.3"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("1.4.4", "1.2.3"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("1.2.4", "1.2.3"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("1.2.4rc0", "1.2.3"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("1.2.4.dev0", "1.2.3"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("1.2.3", "1.2.3"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("1.2.3+git12345678", "1.2.3"));
+    assertTrue(PantsUtil.isCompatiblePantsVersion("1.2.3-git12345678", "1.2.3"));
```

Comment:
I might also add a test for a version like `1.24.0rc1-git2481dab2-md59a1c1708` as well, since we see that in our internal pants repo. I am pretty sure that the parsing mechanism you've developed here is only going to capture the relevant numbers, but in the case that we modify the parsing in the future, it might be good to have a complete example?

Reply:
Added! Thanks